src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
```

Comment:
const x = 40;
let result = days *  x

this example is better for wide project, if something is gonna change you need to fix it in one line "const x = 40;"

Reply:
Transformed 40 into const

  const dailyRate = 40;
  let result = days * dailyRate;